### PR TITLE
feat: web docs added

### PR DESCRIPTION
**Pull request by geeky-abhishek**

### CODE REVIEW

File: use/developer/web/web-channel.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+
+## Home Page
+![The GitBook Logo](../../../media/01.png)
+
+
+
+
+## Fetch bot List
```
Comment (ChakshuGautam) [inline use/developer/web/web-channel.md:27]: Inconsistance whitespace here and at other places as well.

File: use/developer/web/web-channel.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+## About UCI :open_book:
+
+The Unified Communications Interface (UCI) aims to democratize the use of different communication channels such as WhatsApp, Telegram, SMS, email and more for governance use cases through a standard configurable manner that is reusable and scalable across all governance use cases.
```
Comment (ChakshuGautam) [inline use/developer/web/web-channel.md:3]: Is this consistent with the microsite?

File: use/developer/web/web-channel.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+- Ability to connect to any communication channel through any service provider without doing custom changes in the core logic UCI.
+- The UCI ecosystem is independent of external variables like communication channel and service provider powered by XMessage standard.
+- Ability to have a configurable conversation logic for the bot
+- Ability to connect to any database (local or federated) via services
+- Ability to include value added services in the bot interaction flow through Microservices (Internal or External)
+- Ability to create tools on top of UCI APIs to manage Bot configuration, conversations and visualization
+
+
+### About UCI Web Channel
+It's a web client powered by [NextJs](https://nextjs.org/docs) ,[ChatUI](https://www.npmjs.com/package/samagra-chatui) and [Turborepo](https://turbo.build/repo) and using [UCI](https://github.com/samagra-comms) as backend.
```
Comment (ChakshuGautam) [inline use/developer/web/web-channel.md:16]: Let's reverse link to the microsite.

File: use/developer/web/web-channel.md (hunk 2 above)
Comment (geeky-abhishek) [reply use/developer/web/web-channel.md:3]: @ChakshuGautam what do you mean by microsite?

**Pull request merged**